build-tools/create-vsix/create-vsix.csproj

```diff
@@ -7,13 +7,13 @@
     <ProjectGuid>{94756FEB-1F64-411D-A18E-81B5158F776A}</ProjectGuid>
     <OutputType>Library</OutputType>
     <RootNamespace>Xamarin.Android.Sdk</RootNamespace>
-    <AssemblyName>Xamarin.Android.Sdk-OSS</AssemblyName>
+    <AssemblyName>Xamarin.Android.Sdk</AssemblyName>
```

Comment:
I assume that on the commercial build that property will be set to the non-OSS suffixed version?